### ISSUE TITLE
Corners block mouse movement to adjacent monitors

### ISSUE THREAD

**Issue by dnut** [mlutfy/hidetopbar#184]
I have three monitors next to each other. Normally, gnome blocks you from moving your mouse cursor into an adjacent monitor while the hovering over the top bar. With the top bar disabled, I expect to not be blocked from leaving the area that the top bar once was.

After disabling the top bar, there are still about 30 pixels along the left and right edges of my center monitor in the top left and right corners that the cursor cannot pass to access the side monitors. Starting from a side monitor, it can cross those same pixels to enter the center monitor.

I frequently try to click on my leftmost pinned tab in a firefox window on my right monitor, but the mouse is slightly off target and I overshoot it to the left by a few pixels, so I pull the mouse just slightly to the right and click. Instead of hitting the tab, the cursor is blocked by the corner, so I click X on the titlebar of the window open in my center monitor.

**Comment by tuxor1337** [comment]
Closing as duplicate of https://github.com/mlutfy/hidetopbar/issues/151
